src/NUnitFramework/framework/Internal/Tests/TestSuite.cs

```diff
@@ -117,7 +117,7 @@ public TestSuite(TestSuite suite, ITestFilter filter)
                 {
                     if(child.IsSuite)
                     {
-                        TestSuite childSuite = new TestSuite(child as TestSuite, filter);
+                        TestSuite childSuite = ((TestSuite) child).Copy(filter);
```

Comment:
I think our style guide recommends the traditional C# style of no space after direct cast. ReSharper makes Visual Studio start adding the space unless we add `csharp_space_after_cast = false` to our .editorconfig. Ignore the fact that you're seeing some unusual styles in the parts of the file you aren't touching. :D

I'd be grateful if you took this change in the meantime:

```suggestion
                        TestSuite childSuite = ((TestSuite)child).Copy(filter);
```